src/inverseRobot.js

```diff
@@ -7,7 +7,26 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inverseRob = {};
+  const values = Object.values(robot);
+
+  if (values.length === 0) {
+    return inverseRob;
+  }
+
+  for (let i = 0; i < values.length; i++) {
+    for (let k = 1; k < values.length; k++) {
+      if (values[i] === values[k]) {
+        return null;
+      }
+    }
+
+    for (const key in robot) {
+      inverseRob[robot[key]] = key;
+    }
+
+    return inverseRob;
+  }
 }
```

Comment:
Looks too complicated, you can check
if this is (inverseRob[robot[key]]) true then return null, otherwise inverseRob[robot[key]] = key;